daliuge-engine/dlg/drop.py

```diff
@@ -427,28 +427,26 @@ def get_param_value(attr_name, default_value):
             has_app_param = (
                     "applicationArgs" in kwargs and attr_name in kwargs["applicationArgs"]
             )
-            param = default_value
-            if has_component_param and has_app_param:
-                logger.warning(
-                    "Drop has both component and app param %s. Using component param.",
-                    attr_name
-                )
+            # c_param = a_param = default_value
             if has_component_param:
-                param = kwargs.get(attr_name)
+                c_param = kwargs.get(attr_name)
             elif has_app_param:
```

Comment:
May we call these `component_param`, `attribute_param`? It's just that much more readable.